src/Router/Adapter.php

```diff
@@ -159,10 +151,10 @@ function () use ($e, $dataModel, $message) {
     private function handleException(Message $message, \Throwable $e): void
     {
         $onCompleted = function () use ($e, $message) {
-            echo "Message {$message->getRoutingKey()} has been nack".PHP_EOL;
-            echo "Reason: {$e->getMessage()}".PHP_EOL;
-            echo "This occurs in {$e->getFile()} @line {$e->getLine()}".PHP_EOL;
-            $this->logger->error($e->getMessage(), [
+            $this->logger->error("Message {$message->getRoutingKey()} has been nack", [
+                'reason' => $e->getMessage(),
```

Comment:
Things are a little more complex than I expected on this case.
All informations are currently contained in the exception.

The aim here was to print some informations in stdout, then capture the exception with the logger.

But if we strictly replace `echo`s by a logger call with level debug or info, we call twice the logger for the same reason at different levels.

Here the solution can be the usage of [LineFormatter](https://github.com/Seldaek/monolog/blob/master/src/Monolog/Formatter/LineFormatter.php) by the final user and simply delete `reason|file|line` keys.